### PR TITLE
Achievements model

### PR DESCRIPTION
**Pull request by anosek-an**
Three types of achievements introduced - maxed k activities, passed k activities, maxed k subactivities. Test by changing grades and subgrades and looking at the console prints. Current count of passed/maxed activity/subactivity is printed. Outputs will be removed once proper views are introduced.

### CODE REVIEW

**Comment by wyder110** [comment]
You can delete System.out.println() from code 
image

File: src/main/java/upeldev/com/github/upel3/model/achievement/GradeAchievement.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                }
+                else{
+                    grades.remove(grade);
+                }
+
+            break;
+        }
+
+        if(grades.size() >= lowerLimit) isAchieved = true;
+        else isAchieved = false;
```
Comment (wyder110) [inline src/main/java/upeldev/com/github/upel3/model/achievement/GradeAchievement.java]: isAchieved = grades.size() >= lowerLimit; probably would be prettier

**Review by wyder110 — APPROVED** [review]
Seems good to me :)

File: src/main/java/upeldev/com/github/upel3/model/achievement/GradeAchievement.java (hunk 1 above)
Comment (anosek-an) [reply src/main/java/upeldev/com/github/upel3/model/achievement/GradeAchievement.java]: Obviously you're right, thanks. I'll change it right away

**Review by xenoteo — APPROVED** [review]

**Pull request merged**